### DISCUSSION TITLE
Rendering an SVG file

### DISCUSSION THREAD

**Discussion by americanslon** [toddams/RazorLight#497] [Q&A]
I use another library to generate an SVG graphic and write it to a string.
I am then trying to get RazorLight to render it

I tried
`@((MarkupString)ViewBag.QR)`
and
`@ViewBag.QR`

Both code lines just put the svg on the page as a string and don't actually try to render an image.

Is it possible to do what I am trying to do?

**Comment by marcwittke** [comment]
I guess you need

```
@Raw(ViewBag.QR)
```

If you get reddish squiggles: it is a false positive. But you can get rid of it using `@inherits RazorLight.TemplatePage<TModel>`
